### DISCUSSION TITLE
Should the service be named `*Runner` or `*Command`?

### DISCUSSION THREAD

**Discussion by tukusejssirs** [jmcdo29/nest-commander#885] [Ideas]
I understand that the name of the service could be anything, but in the [docs](https://nest-commander.jaymcdoniel.dev/en/features/commander/), you sometimes use `*Runner` (e.g. `TaskRunner`), else you use `*Command` (e.g. `RunCommand`, `FooCommand`).

At the very least, I think you should suggest to use only one or the other, unless there are some rules when to use which one.

What is you suggestion?

**Comment by jmcdo29** [comment]
I generally use `FeatureCommand` and that's what the schematic is set up to create. I think `TaskCommand` sounded weird while I was writing the docs so I called it `TaskRunner` instead. It's all personal convention as to what you like the best though.

**Reply by tukusejssirs** [reply]
Yeah, I too think that using `*Command` sounds better, as the command class extends the runner class, which makes sense to me.

Thanks for the explanation. :pray: Anyway, `TaskCommand` does not sound weird to be, YMMV though. :wink:
